docs/photoniq/est/est-settings.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 60
+title: EST Settings
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for viewing and managing Edge Side Tagging settings.
+
+- **Settings tab** - You can view and manage EST settings in the Settings tab in the web console.
+- **REST API** - Macrometa provides several API calls that allow you to view and manage EST settings.
+
+:::note
+We strongly recommend that you review the setting and consult with Macrometa Support before making any changes.
+:::
+
+## View Settings
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view EST activity in the web console.
+
+1. Log in to your Edge Side Tagging dashboard.
+2. Click **Setting**.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to: [get EST settings](http://localhost:3000/docs/apiEst#/paths/api-est-v1-settings/get).
+
+</TabItem>
+</Tabs>
+
+## Create Settings
+
+REVIEWERS - What does creating settings do? Is this creating a new EST origin? Why can't we do it in the UI?
+
+Use our interactive API Reference with code generation in 18 programming languages to: [create EST settings](http://localhost:3000/docs/apiEst#/paths/api-est-v1-settings/post).
+
+## Update Settings
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view EST activity in the web console.
+
+1. Log in to your Edge Side Tagging dashboard.
+2. Click **Setting**.
+3. Make changes to the setting you want to update and then click **Update**. If you make a mistake and want to undo it before you click **Update**, then click **Revert**.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to: [get EST settings](http://localhost:3000/docs/apiEst#/paths/api-est-v1-settings/patch).
+
+</TabItem>
+</Tabs>
+
+## Delete Settings
+
+REVIEWERS - What does deleting settings do? Is this deleting an EST origin? Why can't we do it in the UI?
+
+Use our interactive API Reference with code generation in 18 programming languages to: [delete EST settings](http://localhost:3000/docs/apiEst#/paths/api-est-v1-settings/delete).
+
+## Edge Side Tagging Settings
+
+Edge Side Tagging provides the following settings. Field names in the web console are different than those in the API responses.
+
+![EST Settings](/img/photoniq/est/est-settings.png)
+
+### Cookies Settings
+
+Define how EST handles cookies.
+
+- **Same Site** - Options are:
+  - **none** - 
+  - **lax** - 
```

Comment:
`lax` will not work with EST

Reply:
Then why is it listed as an option in the API?
<img width="426" alt="image" src="https://github.com/Macrometacorp/docs/assets/105658419/f1b5779c-dd09-4712-8284-dc8594a14704">

Reply:
**cookiesKey**: cookie encryption key
**SameSite**: instructs the browser on when to send the cookie in cross-site requests. EST supports `strict` , `none`
**Secure**: Attribute in cookie settings is a security feature that instructs web browsers to only send the cookie through secure channels
**Origin** : Set origin that will connect to the EST service
**basicAuth** : Configure user password to access EST telemetry API /api/est/v1/telemetry in prometheus format

Reply:
@joubert-macrometa, FYI